contracts/src/v0.8/dev/interfaces/KeeperRegistryInterfaceDev.sol

```diff
@@ -0,0 +1,165 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @notice config of the registry
+ * @dev only used in params and return values
+ * @member paymentPremiumPPB payment premium rate oracles receive on top of
+ * being reimbursed for gas, measured in parts per billion
+ * @member flatFeeMicroLink flat fee paid to oracles for performing upkeeps,
+ * priced in MicroLink; can be used in conjunction with or independently of
+ * paymentPremiumPPB
+ * @member blockCountPerTurn number of blocks each oracle has during their turn to
+ * perform upkeep before it will be the next keeper's turn to submit
+ * @member checkGasLimit gas limit when checking for upkeep
+ * @member stalenessSeconds number of seconds that is allowed for feed data to
+ * be stale before switching to the fallback pricing
+ * @member gasCeilingMultiplier multiplier to apply to the fast gas feed price
+ * when calculating the payment ceiling for keepers
+ * @member minUpkeepSpend minimum LINK that an upkeep must spend before cancelling
+ * @member maxPerformGas max executeGas allowed for an upkeep on this registry
+ * @member fallbackGasPrice gas price used if the gas price feed is stale
+ * @member fallbackLinkPrice LINK price used if the LINK price feed is stale
+ * @member transcoder address of the transcoder contract
+ * @member registrar address of the registrar contract
+ */
+struct Config {
+  uint32 paymentPremiumPPB;
+  uint32 flatFeeMicroLink; // min 0.000001 LINK, max 4294 LINK
+  uint24 blockCountPerTurn;
+  uint32 checkGasLimit;
+  uint24 stalenessSeconds;
+  uint16 gasCeilingMultiplier;
+  uint96 minUpkeepSpend;
+  uint32 maxPerformGas;
+  uint256 fallbackGasPrice;
+  uint256 fallbackLinkPrice;
+  address transcoder;
+  address registrar;
+}
+
+/**
+ * @notice config of the registry
+ * @dev only used in params and return values
+ * @member nonce used for ID generation
+ * @ownerLinkBalance withdrawable balance of LINK by contract owner
+ * @numUpkeeps total number of upkeeps on the registry
+ */
+struct State {
+  uint32 nonce;
+  uint96 ownerLinkBalance;
+  uint256 expectedLinkBalance;
+  uint256 numUpkeeps;
+}
+
+interface KeeperRegistryBaseInterface {
```

Comment:
This is a copy of KeeperRegistryInterface, with the view functions moved separately into `KeeperRegistryInterface` and `KeeperRegistryExecutableInterface`